packages/backend/src/tokens/__tests__/keys.test.ts

```diff
@@ -184,5 +185,38 @@ export default (QUnit: QUnit) => {
         }
       }
     });
+
+    test('cache TTLs do not conflict', async assert => {
```

Comment:
This a really specific test case for the observed bug. I confirmed that this test fails on `main` and passes on this branch. Not sure if it's worthwhile to keep around for a long time with the removal of the timer-based cache eviction. 🤔